tests/models/bart/test_modeling_tf_bart.py

```diff
@@ -225,10 +225,6 @@ def test_decoder_model_past_large_inputs(self):
         config_and_inputs = self.model_tester.prepare_config_and_inputs_for_common()
         self.model_tester.check_decoder_model_past_large_inputs(*config_and_inputs)
 
-    @tooslow
-    def test_saved_model_creation(self):
-        pass
-
```

Comment:
This is still a slow test, and I would like to know if this re-enabled test pass now.

Reply:
It passes!